DataStructures/Stacks/GenericStackList.JAVA

```diff
@@ -0,0 +1,75 @@
+public class Stack <E> {
+	
+	Link head;
+	int cnt;
```

Comment:
Please consider adopting a more comprehensive nomenclature.
```suggestion
	int count;
```